python/src/SymbolicFunction_doc.i.in

```diff
@@ -87,33 +87,64 @@ Examples
 >>> print(f([1, 2]))
 [3,-1]
 
-ExprTk allows to write multiple outputs; in this case, the constructor has a special syntax,
+`ExprTk <http://www.partow.net/programming/exprtk/>`_
+allows to write multiple outputs; in this case, the constructor has a special syntax,
 it contains input variables names, but also output variables names, and formula is a string:
 
 >>> import openturns as ot
 >>> f = ot.SymbolicFunction(['x0', 'x1'], ['y0', 'y1'], 'y0 := x0 + x1; y1 := x0 - x1')
 >>> print(f([1, 2]))
 [3,-1]
 
-ExprTk allows to manage intermediate variables with the 'var' keyword. 
-This is convenient in the situation where several outputs require 
-the same intermediate calculation or if the output is a complex 
-function of the input. 
-In the following example, we compute the 'alpha' variable which is 
-the slope of the river in the flooding example. 
-This slope is then used in the computation of the height 'H'. 
+The following example uses the min and sqrt functions:
+
+>>> formula = 'min(-x1 - x2 - x3 + 3 * sqrt(3), -x3 + 3)'
+>>> limitStateFunction = ot.SymbolicFunction(['x1', 'x2', 'x3'], [formula])
+>>> print(limitStateFunction([1, 2, 3]))
+[-0.803848]
+
+The following example splits the formula into four parts to manage its length:
+
+>>> formula = '15.59 * 1e4 - x1 *x2^3 / (2 * x3^3) *'
+>>> formula += '((x4^2 - 4 * x5 * x6 * x7^2 + '
+>>> formula += 'x4 * (x6 + 4 * x5 + 2 *x6 * x7)) / '
+>>> formula += '(x4 * x5 * (x4 + x6 + 2 *x6 *x7)))'
+>>> input_variables = ['x1', 'x2', 'x3', 'x4', 'x5', 'x6', 'x7']
+>>> limitStateFunction = ot.SymbolicFunction(input_variables, [formula])
+>>> print(limitStateFunction([1, 2, 3, 4, 5, 6, 7]))
+[155900]
```

Comment:
imho this example is unnecessary: showing how to assemble a string is more a Python language example than a relevant openturns example

Reply:
I have no opinion on this topic. This code snippet was proposed by @mbaudin47.

Reply:
I agree on the principle. In theory, every user know that a string can be added with the `+` operator, and that `+=` can concatenate strings. But in practice few beginners realize it and even fewer that a symbolic function can compute non-trivial things. Since this is a situation that we may encounter often, I guess that this might be unnecessary for some, but useful for others.